webui/react/src/routes/index.ts

```diff
@@ -13,13 +14,15 @@ import { ensureAbsolutePath, isFullPath, parseUrl } from 'utils/routes';
  * meaning React will attempt to load the path outside of the internal routing
  * mechanism.
  */
-export interface RouteConfigItem extends RouteProps {
+export interface RouteConfig extends RouteProps {
+  external?: boolean;
```

Comment:
it doesn't show up in the diff. is this used anywhere?

Reply:
oops, that needs to be removed, not used anywhere